terraform/cloudwatch.tf.json.template.py

```diff
@@ -1,34 +1,96 @@
+import json
+
 from azul import (
     config,
 )
 from azul.terraform import (
     emit_tf,
 )
 
-emit_tf(None if config.disable_monitoring else {
-    "resource": [
+
+def dashboard_body() -> str:
+    with open(config.cloudwatch_dashboard_template) as f:
+        body = json.load(f)
+    body = json.dumps(body)
+    replacements = {
+        '542754589326': config.aws_account_id,
+        'us-east-1': config.region,
+        'azul-index-prod': config.es_domain,
+        'azul-indexer-prod-aggregate_retry': config.qualified_resource_name('indexer', suffix='-aggregate_retry'),
+        'azul-indexer-prod-aggregate': config.qualified_resource_name('indexer', suffix='-aggregate'),
+        'azul-indexer-prod-contribute_retry': config.qualified_resource_name('indexer', suffix='-contribute_retry'),
+        'azul-indexer-prod-contribute': config.qualified_resource_name('indexer', suffix='-contribute'),
+        'azul-notifications_fail-prod': config.notifications_queue_name(fail=True),
+        'azul-notifications_retry-prod': config.notifications_queue_name(retry=True),
+        'azul-notifications-prod': config.notifications_queue_name(),
+        'azul-tallies_fail-prod.fifo': config.tallies_queue_name(fail=True),
+        'azul-tallies_retry-prod.fifo': config.tallies_queue_name(retry=True),
+        'azul-tallies-prod.fifo': config.tallies_queue_name(),
+    }
+    # Reverse sorted so that if any keys are substrings of other keys (e.g.
+    # 'foo' and 'foo_bar'), the longer string is processed before the substring.
+    replacements = dict(sorted(replacements.items(), reverse=True))
```

Comment:
```suggestion
    replacements = dict(reversed(sorted(replacements.items())))
```

Yours would be faster but mine is shorter and that's what counts *in this case*.